aws-xray-recorder-sdk-core/src/test/java/com/amazonaws/xray/plugins/ECSPluginTest.java

```diff
@@ -15,39 +15,41 @@
 
 package com.amazonaws.xray.plugins;
 
-import org.junit.Assert;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Rule;
 import org.junit.Test;
```

Comment:
Thanks for the cleanup! Did you try pioneer after changing this import to jupiter? Otherwise, I also wouldn't expect BeforEach to work properly, currently we have both JUnit4 and 5 here I think.

Reply:
Ah I see, yeah let me try converting the whole class to JUnit 5

Reply:
Works now, thanks for the tip!